application/repository/user.py

```diff
@@ -0,0 +1,69 @@
+from application.models.user import User
+
+def get_first_names():
+    users = __get_users_order_by(User.id)
+
+    return [user.first_name for user in users]
+
+def get_telegram_chat_ids(active=False):
+    users = __get_users_order_by(User.id)
+
+    if active:
+        active_users = users.where(User.telegram_notify == 1)
+        users = active_users
+
+    return [user.telegram_chat_id for user in users
+            if user.telegram_chat_id is not None]
+
+def get_telegram_chat_id(first_name):
+    return User.get(User.first_name == first_name).telegram_chat_id
+
+def get_telegram_notify():
+    users = __get_users_order_by(User.id)
+
+    return [user.telegram_notify for user in users]
+
+def get_sms_notify():
+    users = __get_users_order_by(User.id)
+
+    return [user.sms_notify for user in users]
+
+def get_phone_number(chat_id=None, active=0, first_name=None):
+    if first_name and not chat_id and not active:
+        return User.get(User.first_name == first_name).phone_number
+    elif chat_id:
+        return User.get(User.telegram_chat_id == chat_id and
+                        User.sms_notify == active).phone_number
+
+def get_phone_numbers(active=False):
```

Comment:
Maybe we should change active to be integer to stay consistent :D

Reply:
Done :D